src/services/uploader/uploading.controller.test.ts

```diff
@@ -147,20 +147,41 @@ describe('UploadingController', () => {
   });
 
   describe('Start encode', () => {
-    it('Should encode successfully', async () => {
+    it('Should encode successfully when logged in with a hive account', async () => {
       const jwtToken = 'test_jwt_token';
 
-      await linkedAccountsRepository.linkHiveAccount('singleton/starkerz/hive', 'starkerz');
-
       const upload = await uploadingService.createUpload({ sub: 'singleton/starkerz/hive', username: 'starkerz', id: 'test_id' });
 
       process.env.DELEGATED_ACCOUNT = 'threespeak'
 
+      return request(app.getHttpServer())
+        .post('/v1/upload/start_encode')
+        .set('Authorization', `Bearer ${jwtToken}`)
+        .send({
+          upload_id: upload.upload_id,
+          video_id: upload.video_id,
+          permlink: upload.permlink
+        })
+        .expect(201)
+        .then(response => {
+          expect(response.body).toEqual({});
+        });
+    })
+
+    it('Should encode successfully when requesting use of a linked hive account', async () => {
+      const jwtToken = 'test_jwt_token';
+
+      await linkedAccountsRepository.linkHiveAccount('singleton/starkerz/hive', 'sisygoboom');
+
+      const upload = await uploadingService.createUpload({ sub: 'singleton/ned/hive', username: 'sisygoboom', id: 'test_id' });
```

Comment:
Shouldn't the sub be starkerz?
```suggestion
      const upload = await uploadingService.createUpload({ sub: 'singleton/starkerz/hive', username: 'sisygoboom', id: 'test_id' });
```

Reply:
this is different intentionally; it ensures a user can edit a hive post even if a different sub created it.

Reply:
Why would "ned" create an upload for "sisygoboom" if their accounts are not linked? How could that even happen? That authentication should fail. Does "ned" have posting auth over "sisygoboom"? If so, that should be clearly stated, and another test should be created where a Hive account fails to have posting auth over "sisygoboom" and the operation should fail.

Reply:
notice on line 147 I had to use the repository directly in order to bypass the checks in the service.